Functional Monitoring documentation update (4 of 5) (#53)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/afm-run-test-now.adoc

:imagesdir: ../assets/images

> You can run a test before it is next scheduled to run. Doing so does not reset the schedule, which continues to run at previously set intervals.

Reviewer: Simplify?
```suggestion
You can manually run tests outside of the automatic testing schedule without affecting it.
```

== Before You Begin

> Because you are running the test outside of a schedule, you must select the location for the test to run. If you want to use the same location that is used by the schedule, navigate to the monitor detail page and view the location of the schedule.

Reviewer: ```suggestion
Because you are running the test outside of a schedule, you must select the location in which you want it to run. If you want to use the same location as used by the schedule, navigate to the monitor detail page and note that location for future use.
```

== Procedure

> There are two different methods to run a test.

Reviewer: ditto prev. edits to this sentence, previous PRs
